config/chime_keep_gpu_warm.yaml

```diff
@@ -0,0 +1,3783 @@
+#
+##########################################
+---
+########################
+#### Global options ####
+########################
+type: config
+# Logging level can be one of:
+# OFF, ERROR, WARN, INFO, DEBUG, DEBUG2 (case insensitive)
+# Note DEBUG and DEBUG2 require a build with (-DCMAKE_BUILD_TYPE=Debug)
+log_level: info
+num_elements: 2048
+num_local_freq: 1
+num_data_sets: 1
+samples_per_data_set: 49152
+buffer_depth: 10
+baseband_buffer_depth: 282 # 282 = ~34 seconds after accounting for active frames
+vbuffer_depth: 32
+num_links: 4
+timesamples_per_packet: 2
+cpu_affinity: [2,3,8,9]
+block_size: 32
+num_gpus: 4
+link_map: [0,1,2,3]
+
+# Constants
+sizeof_float: 4
+sizeof_int: 4
+sizeof_short: 2
+
+# N2 global options
+num_ev: 4
+# This option now does very little. You probably want to look at
+# visAccumulate:integration_time
+num_gpu_frames: 128
+# Sets the number of sub frames for shorter than ~120ms N2 output
+# Please note this requires changing the number of commands in the
+# GPU section, and the accumulate value for `samples_per_data_set`
+num_sub_frames: 4
+
+# FRB global options
+downsample_time: 3
+downsample_freq: 8
+factor_upchan: 128
+factor_upchan_out: 16
+num_frb_total_beams: 1024
+frb_missing_gains: [1.0,1.0]
+frb_scaling: 0.05 #1.0
+
+# Tracking beams global options
+feed_sep_NS: 0.3048
+feed_sep_EW: 22.0
+num_beams: 10
+num_pulsar_beams: 10
+num_pol: 2
+
+# RFI global options
+sk_step: 256
+rfi_combined: True
+rfi_sigma_cut: 5
+trunc_bias_switch: True
+
+####################
+#### Telescope #####
+####################
+telescope:
+  name: CHIMETelescope
+  require_gps: false # Still run the config if fpga_master is down.
+  query_gps: false
+  query_frequency_map: false
+  require_frequency_map: false
+  allow_default_frequency_map: true
+
+##########################
+#### Dataset manager #####
+##########################
+dataset_manager:
+  # This enables the dataset broker use in GPU kotekan and the RFI server.
+  use_dataset_broker: False
+  ds_broker_host: "10.1.50.11" # recv1
+  ds_broker_port: 12050
+
+##################################
+#### Updatable Config section ####
+##################################
+updatable_config:
+  frb_gain:
+    kotekan_update_endpoint: json
+    frb_gain_dir: /invalid
+  tracking_gain:
+    0:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    1:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    2:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    3:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    4:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    5:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    6:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    7:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    8:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
+    9:
+      kotekan_update_endpoint: json
+      gain_dir: /invalid
```

Comment:
I'm guessing we don't need the extra tracking beam changes?

Reply:
No, and in fact better we not include them here in case I have to revert...

Reply:
Actually we needed them to match the `coco` configuration.